core/block/editor/widget/widget.go

```diff
@@ -14,6 +14,7 @@ const (
 	DefaultWidgetFavorite = "favorite"
 	DefaultWidgetSet      = "set"
 	DefaultWidgetRecent   = "recent"
+	WidgetCollection      = "collection"
```

Comment:
I supposed, `Default` means, that widget is visible in new account by default.  But collection widget should be created by user. So that's why I didn't use `Default` in name. But I can add i